src/aat/resources/application.yml

```diff
@@ -0,0 +1,32 @@
+---
+spring:
```

Comment:
How did you get this to not try to connect to a database? When I did something similar it inherited the main application.yaml config and started connecting to the DB

Reply:
When you specify Bean Classes with SpringBootTest only those are loaded - the application itself is not loaded. Of course, if one of those is Configuration which creates more Beans then those are also created.